backend/translation_orchestrator.py

```
@@ -0,0 +1,172 @@
+                writer.writerow(['Original', 'Translated', 'Status'])
+                # Extract actual segments from the file
+                from scripts.docx_adapter import DocxAdapter
+                adapter = DocxAdapter()
+                try:
+                    segments = adapter.extract_segments(str(input_path))
+                    for seg in segments:
+                        if seg.has_japanese:
+                            writer.writerow([seg.text, f"Translated {seg.text}", "Translated"])
+                        else:
+                            writer.writerow([seg.text, seg.text, "No translation needed"])
+                except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)